src/lib/is_array.js

```diff
@@ -5,6 +5,7 @@
 * This source code is licensed under the MIT license found in the
 * LICENSE file in the root directory of this source tree.
 */
+var Lib = require('../lib');
```

Comment:
You'll need to require `'./is_plain_object.js` to avoid a circular dependency pattern

lib/index -> lib/is_plain_object -> lib/index -> lib/is_array -> lib/index -> lib/is_plain_object